Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerExchangeInformation.ps1

```diff
@@ -72,19 +71,18 @@ function Invoke-AnalyzerExchangeInformation {
         Add-AnalyzedResultInformation @params
     }
 
-    $extendedSupportDate = [System.Convert]::ToDateTime([DateTime]$exchangeInformation.BuildInformation.ExtendedSupportDate,
-        [System.Globalization.DateTimeFormatInfo]::InvariantInfo)
+    $extendedSupportDate = $exchangeInformation.BuildInformation.VersionInformation.ExtendedSupportDate
     if ($extendedSupportDate -le ([DateTime]::Now.AddYears(1))) {
         $displayWriteType = "Yellow"
 
         if ($extendedSupportDate -le ([DateTime]::Now.AddDays(178))) {
             $displayWriteType = "Red"
         }
 
-        $displayValue = "$($exchangeInformation.BuildInformation.ExtendedSupportDate) - Please note of the End Of Life date and plan to migrate soon."
+        $displayValue = "$($exchangeInformation.BuildInformation.VersionInformation.ExtendedSupportDate.ToString("MM/dd/yyyy")) - Please note of the End Of Life date and plan to migrate soon."
```

Comment:
I'm wondering if it makes more sense to display the support date like this: `Tuesday, April 11, 2023`.
Would be easier to understand as the en-US notation might confuse people who normally use the `dd/MM/yyyy` notation.

Reply:
@lusassl-msft does it convert the name of the month if the OS is not English? 

```
$date = get-date
$date.ToString("MMM dd, yyyy")
$date.ToString("MMMM dd, yyyy")
```

If it does, we would also want to force it to be English so when it writes out to the file it should be able to be read by anyone.

Reply:
Tested on German OS:
![image](https://user-images.githubusercontent.com/40993616/206474130-10329f3e-4ab6-48f7-8a96-b3ab55a0a1f4.png)

Reply:
Returns: `Dec 08, 2022`

Reply:
Looks like those are the only two places we display in that format.